### DISCUSSION TITLE
WLED sound reactive music sync

### DISCUSSION THREAD

**Discussion by nextcode4u** [music-assistant/support#3027] [Feature requests and ideas]
I'm looking for a way to sync the audio from music assistant to WLED devices and a way to manage those devices.
 Alternatively LEDFX has been the only way but requires a line level input to a LEDFX instance. The addon for LEDFX is no longer maintained and now requires a separate container or server. 

If WLED was supported or a way to managed RGB/WW devices that allow syncing such as LEDFX would be a great addition to many Home Assistant & Music Assistant setups.

**Comment by bobslaede** [comment]
Maybe in a way to tie the WLED hardware (multiple) to a speaker. That way I can listen to different stuff in different rooms without worrying about the lights too :)

**Comment by rwestnl** [comment]
I've started building this — a working WLED Lights Sync plugin, taking the 2.9
["Light my fire"](https://www.music-assistant.io/blog/2026/06/10/music-assistant-2-9/#light-my-fire)
Hue → Sendspin bridge and doing the same for WLED (both a WLED-processed Audio Sync mode
and an MA-rendered DDP mode with album-cover colours), running on real hardware. Rather
than clutter this request, I've written it up with details, screenshots and a few open
questions for the maintainers here: #6344 

Feedback very welcome 🙂

**Reply by neilenns** [reply]
How can people try this out? I'd love to give it a... spin... with my WLED setup :)

**Reply by rwestnl** [reply]
@neilenns thanks for the interest! Honest heads-up on trying it today: to make it properly user-configurable I had to change things on both sides — the plugin itself and Music Assistant core (the frontend: a sidebar page, per-room effect controls, an in-player effect entry). There's no generic plugin-UI hook yet, so right now it only runs with a patched frontend — not a simple drop-in install.

https://github.com/orgs/music-assistant/discussions/6344#discussioncomment-18358088

**Reply by neilenns** [reply]
Ah yeah, thanks. That's more than I'm willing to bite off :D
